libavformat/imfdec.c

```diff
@@ -984,6 +995,14 @@ static const AVOption imf_options[] = {
         .default_val = {.str = NULL},
         .flags       = AV_OPT_FLAG_DECODING_PARAM,
     },
+    {
```

Comment:
I'm not entirely pleased with this, but it's the easiest way to propagate stream options to a wrapped (inner) demuxer.  This is how it is done in hls.c, for example.